content/templates/implementations.md

```diff
@@ -0,0 +1,89 @@
+- Lottie for VueJS by <a href="https://github.com/chenqingspring/vue-lottie" target="_blank">chenqingspring</a>
+- Lottie for Angular by <a href="https://github.com/chenqingspring/ng-lottie" target="_blank">chenqingspring</a>
+
+##### Android
+
+- Lottie for Android: <a href="https://github.com/airbnb/lottie-android" target="_blank">Official</a>
+- Lottie for React Native <a href="https://github.com/airbnb/lottie-react-native" target="_blank">Official</a>
+- <a href="https://github.com/martijn00/LottieXamarin" target="_blank">Xamarin bindings</a>
+- <a href="https://github.com/bradmartin/nativescript-lottie" target="_blank">NativeScript bindings</a>
+- <a href="https://github.com/m1ga/ti.animation" target="_blank">Appcelerator Titanium bindings</a>
+- Fluttie - Lottie for Flutter <a href="https://github.com/simolus3/fluttie" target="_blank">View</a>
+
+##### iOS
+
+- Lottie for iOS <a href="https://github.com/airbnb/lottie-ios" target="_blank">Official</a>
+- Lottie for React Native <a href="https://github.com/airbnb/lottie-react-native" target="_blank">Official</a>
+
+##### Desktop
+
+- Lottie for Windows <a href="https://github.com/windows-toolkit/Lottie-Windows" target="_blank">View</a>
+- Lottie for UWP by <a href="https://github.com/azchohfi/LottieUWP" target="_blank">azchohfi</a>
+
+##### Multi-Platform
+
+- rLottie by @Samsung <a href="https://github.com/Samsung/rlottie" target="_blank">View</a>
+- QT <a href="https://www.qt.io/blog/2019/03/08/announcing-qtlottie" target="_blank">View</a>
+- Skottie - Lottie Animation Player <a href="https://skia.org/user/modules/skottie" target="_blank">View</a>
+
+<br/>
+
+## Tools
+
+- Lottie Interactivity Library - <a href="https://lottiefiles.com/interactivity" target="_blank">View</a>
+- Lottie JSON Editor - <a href="https://lottiefiles.com/tools/json-editor" target="_blank">View</a>
+- Lottie Web Editor - Editor Lottie Animations online <a href="http://lottiefiles.com/editor" target="_blank">View</a>
+- Sketch2AE - A Sketch plugin to export sketch file to Adobe After Effect <a href="https://github.com/bigxixi/Sketch2AE" target="_blank">Read</a>
+- puppeteer-lottie - Renders Lottie animations via Puppeteer to image, GIF, or MP4 <a href="https://github.com/transitive-bullshit/puppeteer-lottie" target="_blank">View</a>
+- lottie-colorify - Change lottie colors from code <a href="https://github.com/xxmuaddib/lottie-colorify" target="_blank">View</a>
+- lottie-interactive - Custom web element to add interactivity to your Lottie Animations <a href="https://github.com/samuelOsborne/Lottie-interactive" target="_blank">View</a>
+- lottie-toggles - Utility library to easily use the various switch and toggle animations <a href="https://github.com/CoderVishalSehgal/lottie-toggles" target="_blank">View</a>
+- relottie - Lottie processor powered by plugins <a href="https://github.com/lottiefiles/relottie" target="_blank">View</a>
```

Comment:
python-lottie - Library to manipulate Lottie animate and convert between file formats https://gitlab.com/mattbas/python-lottie